pom.xml

```diff
@@ -48,6 +48,7 @@
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
+            <version>1.18.30</version>
```

Comment:
Why?

Reply:
Yeah of course. For some reason, I am not able to run local host for the GauchoRide App or any other previous projects we've done without updating the Lombok version to 1.18.30 otherwise it gives me an error and I can't test my code. So essentially, its there just so I can test on local host and in this case see that I was able to fix the bug for the dropoff and pickup time